unit_conversion/unit_conversion.py

```diff
@@ -348,7 +352,8 @@ class InvalidUnitError(UnitConversionError):
     Exception raised when a unit is not in the Unit conversion database
 
     """
-    def __init__(self, (unit, unit_type) ):
+    def __init__(self, unit_unit_type):
+        (unit, unit_type) = unit_unit_type
```

Comment:
why is this needed? does py3 not unpack tuples in funcition signatures in the same way?

though it doesn't change the API, so no big deal either way.